frontend/server/src/test/java/org/pytorch/serve/util/ConnectorTest.java

```diff
@@ -0,0 +1,117 @@
+package org.pytorch.serve.util;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ConnectorTest {
```

Comment:
@harshbafna Please add tests for the new Metrics port checks as well

Reply:
Done.